src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Services/TransferRequestDomainServiceTests.cs

```diff
@@ -284,6 +302,46 @@ public Task ApproveTransferRequest(long transferRequestId = 0)
             return Sut.ApproveTransferRequest(transferRequestId, TransferSenderUserInfo, Now, default);
         }
 
+
+        public Task GetTransferRequestSummary()
+        {
+            var accountId = Fixture.Create<long>();
+            Sut = new TransferRequestDomainService(new Lazy<ProviderCommitmentsDbContext>(() => DbContext), Logger.Object);
```

Comment:
is there any reason to create a new Sut here?

Reply:
removed